index.html

```diff
@@ -1,319 +1,319 @@
-                    </li>
-                    <li class="p-list__item">
-                        <input class="mktoField" value="yes" id="canonicalUpdatesOptIn" name="canonicalUpdatesOptIn" type="checkbox" />
-                        <label class="mktoLabel mktoHasWidth" for="canonicalUpdatesOptIn">Please send me information about Canonical's products and services.</label>
-                    </li>
-                    <li class="p-list__item"><p>Submitting this form confirms that I have read and agree to <a href="/legal/data-privacy/contact">Canonical&rsquo;s Privacy Notice</a> and <a href="/legal/data-privacy">Privacy Policy</a>.</p></li>
-                    <li class="p-list__item">
-                        <div class="g-recaptcha" data-sitekey="6LfYBloUAAAAAINm0KzbEv6TP0boLsTEzpdrB8if" style="margin: 2rem 0;"></div>
-                    </li>
-                    </ul>
-    
-                    <div class="u-hide">
-                    <h3>Your comments</h3>
-                    <ul class="p-list">
-                        <li class="mktFormReq mktField p-list__item">
-                        <label for="dynamic-forms-comments" class="mktoLabel">What would you like to talk to us about?</label>
-                        <textarea id="dynamic-forms-comments" name="dynamic-forms-comments" rows="5" class="mktoField" maxlength="2000"></textarea>
-                        </li>
-                    </ul>
-                    <input type="hidden" aria-hidden="true" aria-label="hidden field" name="formid" value="1337" />
-                    <input type="hidden" aria-hidden="true" aria-label="hidden field" name="returnURL" value="https://charmed-kubeflow.io/thank-you" />
-                    <input type="hidden" aria-hidden="true" aria-label="hidden field" name="Consent_to_Processing__c" value="yes" />
-                    </div>
-    
-                    <ol class="p-pagination">
-                    <li class="p-pagination__item">
-                        <a class="p-pagination__link--previous p-button" href="">Previous</a>
-                    </li>
-                    <li class="p-pagination__item">
-                        <button type="submit" class="p-button--positive" aria-label="Submit" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Form', 'eventAction' : 'charmed-kubeflow contact-us', 'eventLabel' : '{{product}}', 'eventValue' : undefined });">Let's discuss</button>
-                    </li>
-                    </div>
-                </form>
-                </div>
-            </div>
-            </div>
-        </div>
-    </div>    
-    </div>    
-    <script src="/build/js/dynamic-forms.js"></script>
-    <script type="text/javascript">
-      dynamicForms.setupForm();
-    </script>
-  </body>
-</html>
+<body>
+	<h1>Page content</h1>
+	<p>
+		This is the demo of the interactive forms. Click on the "contact us" button to trigger the interactive form.
```

Comment:
Looks like you indentation on this file is wrong? Should be 2 spaces.

Reply:
yeah I don't know why my editor keeps doing that, I'll see if I can change the default